test/helpers/cilium.go

```diff
@@ -60,34 +75,38 @@ func (c *Cilium) EndpointsGet(id string) *models.Endpoint {
 	return nil
 }
 
-//EndpointSetConfig set to a container endpoint a new config
-func (c *Cilium) EndpointSetConfig(container, option, value string) bool {
+//EndpointSetConfig sets the provided configuration option to the provided
+//value for the endpoint with the provided id.
+func (c *Cilium) EndpointSetConfig(id, option, value string) bool {
 	// on grep we use an space, so we are sure that only match the key that we want.
+
+	logger := c.logCxt.WithFields(log.Fields{"EndpointId": id})
 	res := c.Exec(fmt.Sprintf(
-		"endpoint config %s | grep '%s ' | awk '{print $2}'", container, option))
+		"endpoint config %s | grep '%s ' | awk '{print $2}'", id, option))
+
 	if res.SingleOut() == value {
-		return res.Correct()
+		return res.WasSuccessful()
 	}
 
-	before := c.EndpointsGet(container)
+	before := c.EndpointGet(id)
 	if before == nil {
 		return false
 	}
-	data := c.Exec(fmt.Sprintf("endpoint config %s %s=%s", container, option, value))
-	if !data.Correct() {
-		c.logCxt.Infof("Can't set endoint '%d' config %s=%s", container, option, value)
+	data := c.Exec(fmt.Sprintf("endpoint config %s %s=%s", id, option, value))
+	if !data.WasSuccessful() {
+		logger.Infof("Can't set endoint config %s=%s", option, value)
 		return false
 	}
 	err := WithTimeout(func() bool {
-		status := c.EndpointsGet(container)
+		status := c.EndpointGet(id)
 		if len(status.Status) > len(before.Status) {
 			return true
 		}
-		c.logCxt.Infof("Endpoint '%s' is not regenerated", container)
+		logger.Info("Endpoint is not regenerated")
 		return false
 	}, "Endpoint is not regenerated", &TimeoutConfig{Timeout: 100})
 	if err != nil {
-		c.logCxt.Infof("Endpoint set failed:%s", err)
+		logger.Infof("Endpoint set failed:%s", err)
```

Comment:
change from "Endpoint set failed" to something more descriptive, as this is a bit cryptic.